di-ipv-cimit-stub/lambdas/get-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/getcontraindicatorcredential/domain/cimitcredential/Evidence.java

```diff
@@ -1,11 +1,12 @@
 package uk.gov.di.ipv.core.getcontraindicatorcredential.domain.cimitcredential;
 
 import java.util.List;
+import java.util.SortedSet;
 
-public record Evidence(List<ContraIndicator> contraIndicator, List<String> txn, String type) {
+public record Evidence(List<ContraIndicator> contraIndicator, SortedSet<String> txn, String type) {
     private static final String SECURITY_CHECK = "SecurityCheck";
 
-    public Evidence(List<ContraIndicator> contraIndicator, List<String> txn) {
+    public Evidence(List<ContraIndicator> contraIndicator, SortedSet<String> txn) {
```

Comment:
I assume like the issuers you've checked that it's correct to dedupe these? I still kinda prefer having plain old `List` on the model classes (since it represents a JSON structure), but appreciate it's convenient to reflect the deduplication semantics.

Reply:
Yeah, I've had a chat with @michaelabenyohai about this. The txn values at the evidence level are a deduplicated list of those in the CI items themselves.

And yeah, it's not a direct representation but it saved some logic, and it expresses the intent some.